recipe/run_conda_forge_build_setup_win

```diff
@@ -8,5 +8,5 @@ conda install -n root --yes --quiet jinja2 anaconda-client
 
 # see https://github.com/conda-forge/staged-recipes/pull/750#issuecomment-225165530
 conda install -n root --yes --quiet conda-build=1.20.1
-
+if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install -n root --yes --quiet conda-build=1.20.0
```

Comment:
Sorry, could have been clearer. Was meaning this [comment]( https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/646cd541768932c480724b6d41801bd973b434f6/recipe/run_conda_forge_build_setup_win#L9-L11 ).